### ISSUE TITLE
CC-282.1 (Level AA) _ Modify Modules _ Button _ Truncation of labels of buttons

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3157]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	Medium
Affected UI	Button
Affected Check Point:	ACC-282.1
Short text:	ACC-282.1 (Level AA) _ Modify Modules _ Button _ Truncation of labels of buttons

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Now add the Device with Text Responsive settings of width 320 CSS pixels and height 700 CSS pixels from the ‘Developers tool’ from the ‘Chrome’ Browser and apply the settings.
•	Now refresh the page after the settings are applied.
•	Activate ‘Edit’ tab from main region.

3. Issue Type: Manual
Observed Behavior:
On resizing the screen width to 320 CSS pixels, it is observed that there is truncation of text of buttons e.g. ‘Form/YAML’ segmented buttons and ‘Save’ button within ‘Edit’ tab of the page which confuses the user.
Screenshot:
Default:
 

After text resize to 320 css:
               

Expected Behavior:
Truncation should be avoided, when the content of the screen is displayed on a width of 320 pixel. Users with a small viewport, regardless of whether they use a small device or a considerable zoom, must be able to consume text as well as information given by other means without the need to scroll in reading direction while consuming one piece of information.
Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.
________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[ACC-282.1 (Level AA) _ Modify Modules _ Button _ Truncation of labels of buttons.docx](https://github.com/user-attachments/files/16660289/ACC-282.1.Level.AA._.Modify.Modules._.Button._.Truncation.of.labels.of.buttons.docx)

[Issue processing-09.2021 (002).docx](https://github.com/user-attachments/files/16660290/Issue.processing-09.2021.002.docx)
